packages/muon/tokens/theme/font.json

```diff
@@ -55,6 +55,10 @@
           "description": "Weight of the font used for headings.",
           "value": "600"
         }
+      },
+      "lineLength": {
```

Comment:
Is this default lineLength?

Reply:
Yeah, that's the intention. This can be applied to the typography and to other elements within components.

Like here in the cta: https://github.com/centrica-engineering/muon/blob/main/packages/muon/components/cta/src/cta-styles.css#L86

And in the inputter: https://github.com/centrica-engineering/muon/blob/feature/inputter-styles/packages/muon/components/inputter/src/inputter-styles.css#L11